selfdrive/car/hyundai/values.py

```diff
@@ -30,7 +30,7 @@ class CAR:
   KONA_EV = "HYUNDAI KONA ELECTRIC 2019"
   SANTA_FE = "HYUNDAI SANTA FE LIMITED 2019"
   SONATA = "HYUNDAI SONATA 2020"
-  SONATA_2019 = "HYUNDAI SONATA 2019"
+  SONATA_LF = "HYUNDAI SONATA 2018-2019"
```

Comment:
```suggestion
  SONATA_LF = "HYUNDAI SONATA 2019"
```

We usually don't change the car name after it's been in there for a while. This string ends up in our backend.